appveyor.yml

```diff
@@ -168,19 +169,27 @@ before_build:
     - cmd: b2 -j2 --toolset=msvc-14.0 --layout=versioned %ALPAKA_BOOST_B2% architecture=x86 address-model=%BOOST_ADDRESS_MODEL% variant=%BOOST_VARIANT% link=static threading=multi runtime-link=shared define=_CRT_NONSTDC_NO_DEPRECATE define=_CRT_SECURE_NO_DEPRECATE define=_SCL_SECURE_NO_DEPRECAT define=BOOST_USE_WINFIBERS --stagedir="%ALPAKA_B2_STAGE_DIR%"
 
     #-------------------------------------------------------------------------------
+    # Install TBB
+    - cmd: set TBB_ROOT_DIR=C:\projects\tbb
+    - cmd: git clone -b master --recursive --single-branch --depth 1 https://github.com/intel-tbb/intel-tbb.git %TBB_ROOT_DIR%
```

Comment:
This clone will not be enough, as the prebuild dynamic libraries will not be installed.
You can remove this last line and I can add the correct installation later (not necessarily in this Pull Request).